public/manifest.json

```diff
@@ -1,25 +1,36 @@
 {
-  "short_name": "Oasis Wallet",
```

Comment:
Maybe the old manifest.json can be kept and renamed to web-app-manifest.json to reduce confusion